ci/scan-image.yml

```
@@ -73,15 +73,22 @@ run:
       ADDITIONAL_ARGS="${ADDITIONAL_ARGS} --detailed-results"
     fi
 
+    export EXIT_CODE
```

You don't have to export it here, you can just do
```suggestion
    EXIT_CODE
```
or
```suggestion
    local EXIT_CODE
```